code/modules/clothing/suits/armor.dm

```diff
@@ -70,23 +70,12 @@
 	icon_state = "hostrench"
 	item_state = "hostrench"
 	flags_inv = 0
+	ignore_suitadjust = 0
+	action_button_name = "Open/Close Trenchcoat"
+	armor_adjust = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
 
-	verb/toggle()
-		set name = "Toggle Trenchcoat Buttons"
-		set category = "Object"
-
-		if(!usr.canmove || usr.stat || usr.restrained())
-			return 0
-		if(icon_state == "hostrench")
-			icon_state = "hostrench_button"
-			item_state = "hostrench_button"
-			usr<< "You button the [src]."
-		else
-			icon_state = "hostrench"
-			item_state = "hostrench"
-			usr<< "You unbutton the [src]."
-
-		usr.update_inv_wear_suit()
+/obj/item/clothing/suit/armor/hos/alt/attack_self(var/mob/user)

```

Comment:
@Fox-McCloud Instead of having an attack_self for every single one of these objects that has ignore_suitadjust to call the adjustment proc, would it be better if I just had one attack_self for all suits up in clothing.dm that checked to see if ignore_suitadjust was set to 0 before calling adjustsuit(user)?

I know it'd cut down on the amount of code by a fair bit, but IDK if it'll have a negative impact